### PR TITLE
Add ADR for SwiftSoup to explain our reasoning

### PR DESCRIPTION
**Pull request by joshuatbrown**
## Issues covered
None

## Description
Just wanted to document why we're using SwiftSoup as opposed to any of the alternatives.

### CODE REVIEW

File: doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Authors: Josh Brown
+
+## Status
+
+Accepted
+
+## Context & Problem Statement
+
+In order to display a video in portrait or landscape orientation based on its [Open Graph](https://ogp.me/) metadata, we need to parse the HTML document containing the video. The Open Graph metadata contains properties that specify the width and height of the video, and similar properties for an image that may be used as a cover image
```
Comment (pelumy) [inline doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md:13]: Looks like a fullstops is missing at the end of this sentence.

**Review by pelumy — APPROVED** [review]

File: doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md (hunk 1 above)
Comment (joshuatbrown) [reply doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md:13]: Good eye! Thanks for catching this!

Comment (joshuatbrown) [reply doc/adr/0007-use-swiftsoup-to-parse-open-graph-metadata.md]: Fixed!

**Review by mplorentz — APPROVED** [review]
Thanks for the writeup!

**Pull request merged**